### DISCUSSION TITLE
Option to allow Enterprise Managed Users to create issues on public repositories

### DISCUSSION THREAD

**Discussion by dlo1** [community/community#14459] [Enterprise]
We're using Enterprise Managed Users in our organization for full integration with Azure AD SSO. The drawback for us is that EMUs are almost fully disconnected from the public and there are many limitations in place.

It would be great if there was an option that can be enabled in the Enterprise which allows Enterprise Managed Users to create issues on, comment on and fork public repositories. At the moment the only alternative is to create a private GitHub account and use that one as a separate login.

**Comment by macphil** [comment]
EMUs can also not participate here:
![image](https://user-images.githubusercontent.com/7562031/179493480-a1855623-80d1-4ecd-a1a0-e3bbb97903fa.png)

**Reply by JinsengIT** [reply]
I was going to create my own question about this, but I found this one.  We're in the same situation.  Entra ID SSO is a requirement for our systems, so we configured EMU.  Now this means that our users cannot collaborate with our customers.  I'm going to have to tell our end user to create a personal account and use that.  This is also against our normal operating procedures.  I'm amazed that over 3 years later this is still a problem.
